nipype/interfaces/afni/utils.py

```diff
@@ -491,6 +701,76 @@ class Copy(AFNICommand):
     input_spec = CopyInputSpec
     output_spec = AFNICommandOutputSpec
 
+class DotInputSpec(AFNICommandInputSpec):
+    in_files = traits.List(
+        (File()),
```

Comment:
The check for file existence isn't currently compatible with AFNI's sub-brick selector, until I get around to addressing that, I'd like to leave off the existence requirement for this interface.